eda_analysis/eda_analysis.py

```diff
@@ -0,0 +1,14 @@
+def describe_na_values(dataframe):
+    '''
+    describes the na_values in an input pandas dataframe as a 2d array of 1's and 0's.
+    Parameters
+    ----------
+    dataframe
```

Comment:
I think you followed Scipy docstring style. It says to also explicitly specify the data type of parameter after its name.
dataframe: pandas.DataFrame. It would be good if you could add that